### DISCUSSION TITLE
Security documentation?

### DISCUSSION THREAD

**Discussion by JRBeckstrom** [quarto-dev/quarto-cli#1915] [Q&A]
I want to install Quarto on a corporate network and need to do a risk assessment of Quarto. Fair enough, as I actually have to install something. However, and to be honest, I find the documentation regarding security very limited. Are there any "notes on security" documented anywhere?  For example something like [this](https://pandoc.org/MANUAL.html#a-note-on-security)?

**Comment by melissavanbussel** [comment]
Are you able to install RStudio on your corporate network? It might be easier to get approval for installing a newer version of RStudio than it is to get approval for Quarto as a standalone. For example:

<img width="482" alt="image" src="https://user-images.githubusercontent.com/28693927/194972301-575b4192-f190-44cf-8d7b-1e97403321f2.png">

https://quarto.org/docs/tools/rstudio.html
